behat-tests/features/bootstrap/FeatureContext.php

```diff
@@ -369,30 +402,21 @@ public function assertRegionElementHasClass($selector, $region, $class) {
   }
 
   /**
-   * @When I wait for messagebox to close
```

Comment:
Then can I also delete iWaitSeconds()? This is a popular step, but the only good use of it was in debugging, and we already have a very nice ```@Then I break``` from DrupalContext.php.